src/mdformat/_cli.py

```diff
@@ -198,6 +202,11 @@ def make_arg_parser(
     parser.add_argument(
         "--check", action="store_true", help="do not apply changes to files"
     )
+    parser.add_argument(
+        "--no-validate",
+        action="store_true",
```

Comment:
Thanks Kyle! A drive-by comment: let's use `dest="validate"` here so we get a non-negated option in the TOML file.
```suggestion
        action="store_false",
        dest="validate",
```